DownloadManager/Plugin.DownloadManager.Abstractions/IDownloadFile.cs

```diff
@@ -79,5 +79,11 @@ public interface IDownloadFile : INotifyPropertyChanged
         /// </summary>
         /// <value>The total bytes written.</value>
         float TotalBytesWritten { get; }
+
+        /// <summary>
+        /// Gets or sets wether this file can use Mobile network to download.
+        /// </summary>
+        /// <value>Indicates wether this file can use Mobile network for downloading</value>
+        bool MobileNetworkAllowed { get; set; }
```

Comment:
The download should restart if we change the MobileNetworkAllowed property while a download is running. See the DownloadFileImplementation of the respective platforms.